packages/blocks/src/blocks/core-image.tsx

```diff
@@ -248,17 +249,13 @@ const getImageProps = (
  *
  * @return Whether the lightbox is enabled.
  */
-//@ts-ignore -- Stubbed until lightbox support is fixed.
 const isLightboxEnabled = ( lightbox?: string | null ): boolean => {
```

Comment:
Update the doc-block to explicity note that if enabled the `<LightboxOverlay>` component will added in the template render, and a `@todo` to colocate the overlay here and portal it where needed